src/v2/Apps/Collect/Routes/Collection/index.tsx

```diff
@@ -139,6 +141,15 @@ export class CollectionApp extends Component<CollectionAppProps> {
                     artworksConnection.aggregations as SharedArtworkFilterContextProps["aggregations"]
                   }
                   onChange={updateUrl}
+                  onArtworkBrickClick={artwork => {
+                    tracking.trackEvent(
+                      clickedMainArtworkGrid({
+                        contextPageOwnerType: OwnerType.collection,
```

Comment:
We should include the `contextPageOwnerSlug` and `contextPageOwnerId` here as well.

Reply:
Yup, good call.